ui/webui/src/components/installation/InstallationProgress.jsx

```diff
@@ -80,13 +81,16 @@ export const InstallationProgress = ({ onAddErrorNotification, idPrefix, isBootI
                             }
                             if (message) {
                                 setStatusMessage(message);
+                                refStatusMessage.current = message;
                             }
                         });
                         taskProxy.addEventListener("Failed", () => {
                             setStatus("danger");
                         });
                         taskProxy.addEventListener("Stopped", () => {
-                            taskProxy.Finish().catch(onAddErrorNotification);
+                            taskProxy.Finish().catch(onCritFail({
+                                context: cockpit.format(_("Installing the system: $0"), refStatusMessage.current),
```

Comment:
Why don't you use the 'message' directly here? I guess I am missing something?

Reply:
Message is not available here. I didn't want to use statusMessage because I would have to add it to the hook dependencies and the hook (getting tasks, registering handlers, starting the queue) would be called on every message change?